gradle.properties

```diff
@@ -23,4 +23,4 @@
 
 org.gradle.daemon=true
 org.gradle.parallel=true
-org.gradle.jvmargs=-XX:MaxPermSize=1024m
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
Perfect, I've created a ticket for it: https://jira.catrob.at/browse/CAT-2089
and removed it here.